hera_cal/reflections.py

```diff
@@ -225,34 +157,63 @@ def model_reflections(self, dly_range, keys=None, data='clean', edgecut=0, taper
 
             # find reflection
             echo("...Modeling reflections in {}, assigning to {}".format(k, rkey), verbose=verbose)
-            (eps, amp, delays, phs, inds, sig, dfft,
-             dly_arr) = fit_reflection(data[k], dly_range, freqs, edgecut=edgecut, taper=taper,
-                                       zero_pad=zero_pad, full_freqs=self.freqs, fthin=fthin, **win_kwargs)
-            self.epsilon[rkey] = eps
-            self.amps[rkey] = amp
-            self.phs[rkey] = phs
-            self.delays[rkey] = delays
-            self.peak_ratio[rkey] = sig
-            self.dfft[k] = dfft
-
-    def write_reflections(self, output_calfits, input_calfits=None, overwrite=False, add_to_history=''):
+
+            # fit for reflection delays and amplitude
+            amp, dly, inds, sig = fit_reflection_delay(self.dfft[k], dly_range, self.delays, return_peak=False)
+
+            # make significance cut
+            if np.max(sig) < ref_sig_cut:
```

Comment:
Wouldn't it be more sensible to simply write the amplitude, delay, and phase as 0? I get the desire to not have to solve for the phase if that's the slow part, but I think it's weird to ask it to find reflections for some number of antennas and for the code to silently only solve for some antennas.

That way you can record ref_significance rather than losing it and also fill out ref_eps.

Reply:
yeah that's a good idea